unsideline-models/scan_with_start_time_end_time_request.go

```diff
@@ -0,0 +1,8 @@
+package unsideline_models
+
+type ScanWithStartTimeEndTimeRequest struct {
```

Comment:
Same as mentioned earlier. All related models can reside in same file. Probably like `modes/unsideline.go` and `models/sideline.go`

Reply:
will change this